build/shadowsocks_config.d.ts

```diff
@@ -1,3 +1,4 @@
+/// <amd-module name="outline_shadowsocksconfig/src/shadowsocks_config" />
```

Comment:
in some repos we do some stripping of this thing. 

We typically use an `npm_package` target so that the thing that's distributed on npm isn't identical to the sources. And that way we don't check in any generated artifacts, which are confusing if the repo is in a state where the sources are updated but the generated artifacts are not

Reply:
On our specific use case, the reason for this repo is to share code between outline-server and outline-client. We don't really publish this as a public npm, but we do use it as a npm dependency, using a github specification.

I'm converting outline-server to Bazel, and having this repo as Bazel will probably help, since I'm still having issues with ShadowsocksConfig there.

Reply:
How do I strip the /// header? Is that an option in the build rule?